app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelService.java

```diff
@@ -19,7 +19,16 @@ public CellTypeWheelService(CellTypeWheelDao cellTypeWheelDao) {
     }
 
     public ImmutableSet<ImmutablePair<ImmutableList<String>, String>> search(String searchTerm) {
-        return cellTypeWheelDao.facetSearchCtwFields(searchTerm)
+        return explodeTuples(cellTypeWheelDao.facetSearchCtwFields(searchTerm));
```

Comment:
If you could refactor the `facetSearchCtwFields`, then please also change this line of code. Thanks